test/models/order_test.rb

```diff
@@ -1,7 +1,12 @@
-require 'test_helper'
+require "test_helper"
 
 class OrderTest < ActiveSupport::TestCase
   test "when Order exists" do
     assert Order
   end
+
+  test "response from bakery if order is done" do
+    orders_for_delivery = Order.completed_orders
```

Comment:
It wont produce the goods we are testing because we are just getting all of the baked_jobs that have been produced but I understand your point. I can stub the request.